src/osd/PeeringState.cc

```diff
@@ -3083,6 +3103,34 @@ void PeeringState::proc_replica_log(
   peer_missing[from].claim(std::move(omissing));
 }
 
+void PeeringState::calc_min_last_complete_ondisk() {
+  eversion_t min = last_complete_ondisk;
+  ceph_assert(!acting_recovery_backfill.empty());
```

Comment:
can probably be moved above the previous line (i.e. placed as the 1st line of the function)